modules/assim.batch/NAMESPACE

```diff
@@ -1,11 +1,13 @@
-# Generated by roxygen2: do not edit by hand
+# Generated by roxygen2 (4.0.2): do not edit by hand
```

Comment:
@istfer I think you need to upgrade your version of roxygen. Newer versions don't embed the version number in every single file, which means fewer files need to be modified whenever you rebuild the documentation. Could you do that and then update the pull request, it should greatly reduce the number of files changed.